### PR TITLE
remove the max length on richtext forms

### PR DESCRIPTION
**Pull request by MilanBarande**

### CODE REVIEW

File: assembl/static2/tests/unit/components/debate/thread/__snapshots__/answerForm.spec.jsx.snap [hunk 1]
```diff
@@ -102,7 +102,7 @@ exports[`AnswerForm component should render a answer form 1`] = `
             }
           }
           handleInputFocus={[Function]}
-          maxLength={3000}
+          maxLength={0}
```
Comment (ayyazdaniaryan) [inline assembl/static2/tests/unit/components/debate/thread/__snapshots__/answerForm.spec.jsx.snap]: `NO_BODY_LENGTH` instead. Same thing, but let's use the constants we've created.

File: assembl/static2/js/app/components/debate/thread/answerForm.jsx [hunk 2]
```diff
@@ -154,7 +153,6 @@ export class DumbAnswerForm extends React.PureComponent<Props, State> {
                 <RichTextEditor
                   editorState={this.state.body}
                   handleInputFocus={this.handleInputFocus}
-                  maxLength={BODY_MAX_LENGTH}
```
Comment (ayyazdaniaryan) [inline assembl/static2/js/app/components/debate/thread/answerForm.jsx]: Is this required for Bright Mirror and/or admin views as well? If so, then you should remove the logic of `maxLength` in RichTextEditor class.

Comment (ayyazdaniaryan) [reply assembl/static2/js/app/components/debate/thread/answerForm.jsx]: Not sure if it is part of the scope of the bug or not

Comment (MilanBarande) [reply assembl/static2/js/app/components/debate/thread/answerForm.jsx]: I think it's just the thread

Comment (MilanBarande) [reply assembl/static2/js/app/components/debate/thread/answerForm.jsx]: Update: it's everywhere :--) will remove

File: assembl/static2/tests/unit/components/debate/thread/__snapshots__/answerForm.spec.jsx.snap (hunk 1 above)
Comment (MilanBarande) [reply assembl/static2/tests/unit/components/debate/thread/__snapshots__/answerForm.spec.jsx.snap]: mmh I didn't notice there was a defaultProps to 0, weird choice. Anyway, this is going

**Pull request merged**